src/tests/examples/example_test.py

```diff
@@ -37,6 +37,13 @@ def find_integrity_path(output_dir):
     assert len(op) == 1
     return op[0]
 
+def find_values_hourly_path(output_dir):
```

Comment:
This function could be used for a new test. We can add to parameters : 
- area 
- year

Reply:
Done